### DISCUSSION TITLE
Run osu-stable through wine wayland

### DISCUSSION THREAD

**Discussion by Damnjelly** [fufexan/nix-gaming#209] [Q&A] [answered]
Currently osu-stable runs through a wine xwayland session. This has the annoying side-effect of capping the framerate of osu to the monitors refresh rate. Playing osu-stable on anything sub 300 fps makes the game unplayable due to input lag. Is it possible to make osu-stable use one of the wine wayland packages? (e.g. pkgs.wineWowPackages.wayland or pkgs.wineWowPackages.waylandFull)

I'm using Niri with xwayland-satellite as my xwayland session

**Comment by fufexan** [comment, marked as answer]
The current patches that we use don't compile on newer wine versions. I'd like to get osu to run on https://github.com/whrvt/umubuilder/releases/tag/proton-osu-9-5 eventually, but I don't have time to package it.
